boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -184,6 +187,19 @@ public void processOpts() {
                 log.info("  (you can select the angular version by setting the additionalProperty foundationVersion)");
             });
 
+        processOpt(SPEC_VERSION,
+            value -> {
+                if(!Strings.isEmpty(value)) {
```

Comment:
Looking at the build checks, seems there is something wrong with this function syntax? - https://github.com/Backbase/backbase-openapi-tools/runs/8110552249?check_suite_focus=true

Reply:
Yeah you're right. I should've used the StringUtils